packages/navi/build.sh

```diff
@@ -0,0 +1,9 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/denisidoro/navi
+TERMUX_PKG_DESCRIPTION="An interactive cheatsheet tool for the command-line"
+TERMUX_PKG_LICENSE="Apache-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=1.16.0
```

Comment:
Typo in version? Perhaps there should be [2.16.0](https://github.com/denisidoro/navi/releases/tag/v2.16.0) because with current value Github Actions fail to download source archive.